### PR TITLE
EDITORS.md: list of current and former specification editors

### PR DESCRIPTION
**Pull request by ralfhandl**
- [x] Create EDITORS.md
- [x] Use it in BUILD.sh for 3.0.4 and 3.1.1

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
+# OpenAPI Specification Editors
```
Comment (earth2marsh) [inline MAINTAINERS.md:1]: This list has served as the record of TSC membership, though I think that might be made explicit? By adding this title, it also changes the framing. For example, while TSC members are tasked with evolving the spec, that includes managing issues, encouraging community, etc.

File: MAINTAINERS.md [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
+
 ## Active
 * Darrel Miller [@darrelmiller](https://github.com/darrelmiller)
 * Jeremy Whitlock [@whitlockjc](https://github.com/whitlockjc)
 * Marsh Gardiner [@earth2marsh](https://github.com/earth2marsh)
 * Ron Ratovsky [@webron](https://github.com/webron)
 
-## Provisional
```
Comment (earth2marsh) [inline MAINTAINERS.md:7]: Provisional does refer back to the governance processes. What do we lose by dropping it here?

File: MAINTAINERS.md [hunk 3]
```diff
@@ -1,10 +1,13 @@
...
+## Trainee
```
Comment (earth2marsh) [inline MAINTAINERS.md:9]: What makes Trainee better here? I'm not against it, just trying to understand why this word in particular.

**Review by earth2marsh — COMMENTED** [review]
Historically this has been the TSC current and former membership list. Perhaps we should make that explicit? Consider whether "contributors.md" is a more appropriate place to recognize the impact that people like Henry make and/or carve out a different section to explain non-TSC maintainer roles?

**Comment by ralfhandl** [comment]
We currently have
* MAINTAINERS.md: both TSC membership and OAS authors
* CONTRIBUTORS.md: people with more than 25 pull requests

Now TSC members and OAS authors are becoming non-identical groups of people and we may need three documents.

How about adding an EDITORS.md and use that (from now on) in the build script?

File: MAINTAINERS.md (hunk 1 above)
Comment (ralfhandl) [reply MAINTAINERS.md]: Added title to express original purpose of TSC membership

File: MAINTAINERS.md (hunk 3 above)
Comment (ralfhandl) [reply MAINTAINERS.md]: Reverted, added separate EDITORS.md

File: EDITORS.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+## Active
+* Darrel Miller [@darrelmiller](https://github.com/darrelmiller)
+* Henry Andrews [@handrews](https://github.com/handrews)
+* Jeremy Whitlock [@whitlockjc](https://github.com/whitlockjc)
+* Lorna Mitchell [@lornajane](https://github.com/lornajane)
+* Marsh Gardiner [@earth2marsh](https://github.com/earth2marsh)
+* Miguel Quintero [@miqui](https://github.com/miqui)
+* Mike Kistler [@mikekistler](https://github.com/mikekistler)
+* Ron Ratovsky [@webron](https://github.com/webron)
+* Ralf Handl [@ralfhandl](https://github.com/ralfhandl)
```
Comment (earth2marsh) [inline EDITORS.md:12]: Ralf should come before Ron to preserve the alphabetical order! :)

**Review by earth2marsh — DISMISSED** [review]
Thanks for the changes--this looks good to me (other than the minor suggested fix). Ideally we can get another +1, but if not, let's keep moving!

**Review by miqui — APPROVED** [review]

**Review by lornajane — APPROVED** [review]

**Comment by earth2marsh** [comment]
Regarding the discussion in today's call, @ralfhandl I believe we can rename EDITORS.md > TSC.md without issue, as the openapis.org governance menu links to the MAINTAINERS.md (and fun fact, to the old branch name, not `main`!). So once we do the rename, we can work to update the website asynchronously. 

If that makes sense to you, LMK, and I'll make an issue and submit that change!

**Pull request merged**